src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -186,5 +186,10 @@
         "dotnetRuntimeMetrics": true,
         "httpMetrics": true,
         "systemMetrics": true
+    },
```

Comment:
nit: maybe we can put this right below cosmos configurations?

Reply:
That makes total sense - so add after the "CosmosDb" section, correct?

Reply:
Maybe after line 157->"DataStore": "CosmosDb",?

Reply:
OK, then I'll make it like this then.
    "DataStore": "CosmosDb",
    "KeyVault": {
        "Endpoint": null
    },
    "SqlServer": {
        "SqlCommand": {
            "CommandTimeout": 30
        }
    },